routes/matches/matches.js

```diff
@@ -606,12 +645,14 @@ router.get("/:match_id", async (req, res, next) => {
       "FROM `match_pause` " +
       "WHERE match_id = ? ";
     let matchPauseInfo = await db.query(sql, matchId);
-
+    if (!matchPauseInfo.length) {
```

Comment:
Don't do this as connections will drop immediately instead of waiting for event changes.